.travis.yml

```diff
@@ -11,14 +16,26 @@ before_deploy:
   - echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust
 
 deploy:
-  -
+  - provider: script
     skip_cleanup: true
-    provider: script
     script: /bin/bash .travis/deploy.sh
     on:
       tags: true
       condition: "$TRAVIS_TAG =~ ^v([[:digit:]]+\\.)+[[:digit:]]+(-[[:digit:]]+)?(-SNAPSHOT)?$"
 
+before_install:
+  - curl -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz > oc-cli.tar.gz
+  - tar -xzf oc-cli.tar.gz
+  - sudo mv ./openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc /usr/local/bin
+
 after_success:
   - bash <(curl -s https://codecov.io/bash) -f '!*datasafe-business*' -F unittests
   - bash <(curl -s https://codecov.io/bash) -f '*datasafe-business*' -F e2e_tests
+
+  - PROJECT_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+  - oc login https://openshift.adorsys.de:443 --username=$OPENSHIFT_USER --password=$OPENSHIFT_PASSWORD
+  - docker login -u $(oc whoami) -p $(oc whoami -t) https://openshift-registry.adorsys.de
+  - docker build -t datasafe-rest-service --build-arg JAR_FILE=datasafe-rest-impl-$PROJECT_VERSION.jar ./datasafe-rest-impl
```

Comment:
Add image version based on commit ID to docker image tag, latest is ambiguous thing

Reply:
ok